src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationClientFactory.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
```

Comment:
```suggestion
﻿// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.
//
﻿using System;
```

Missed the copyright :)

Reply:
Thanks done